### DISCUSSION TITLE
Cached Organization Avatars do Not reload

### DISCUSSION THREAD

**Discussion by CuzImBisonratte** [community/community#39832] [Mobile] [answered]
## Problem
When navigating to the "Organizations"-Tab, in the List view Cached Avatars do not Reload.  

## I tried
- Reload Gesture (Sliding down from the Top of the View)
- Restarting the App

## What happened
Yesterday I updated many Avatars of my Organizations.  
On Desktop they changed immediately.  
On my phone they changed on app-restart.  
On my iPad they haven't changed at all...  

## Images
![image](https://user-images.githubusercontent.com/50182241/203540527-7243728d-28a4-4d37-b48f-44f867cdd17a.png)
![image](https://user-images.githubusercontent.com/50182241/203540784-8897be57-be8a-4883-9237-c16f5deecf65.png)

<details>
<summary>ℹ️ </summary>
GitHub for iOS, Version 1.90.0 (84076705)
Device: iPad Air 5 (iOS 16.1)
</details>

**Comment by taki-on** [comment, marked as answer]
Thank you for reporting the bug! We will triage it and figure out a fix soon!
